data/db-schema/src/main/resources/net/corda/db/schema/vnode-vault/migration/vnode-vault-creation-v5.1.xml

```diff
@@ -6,5 +6,11 @@
         <addColumn tableName="vnode_member_info">
             <column name="is_deleted" type="BOOLEAN" defaultValueBoolean="false"/>
         </addColumn>
+
+        <createTable tableName="vnode_persistence_request_id">
+            <column name="request_id" type="NVARCHAR(255)">
```

Comment:
Just passing by, but I wonder if NVARCHAR(36) would be suffice as I think a UUID fits in it?

Reply:
We only store the `UUID.toString()` in it. Will take a look to see if we can shorten the DB field.